programs/mango-v4/src/accounts_ix/openbook_v2_cancel_order.rs

```diff
@@ -18,17 +15,15 @@ pub struct OpenbookV2CancelOrder<'info> {
         mut,
         has_one = group,
         constraint = account.load()?.is_operational() @ MangoError::AccountIsFrozen
-        // owner is checked at #1
+        // authority is checked at #1
     )]
     pub account: AccountLoader<'info, MangoAccountFixed>,
 
     pub authority: Signer<'info>,
 
-    #[account(
-        mut,
-        constraint = open_orders.load()?.market == openbook_v2_market_external.key()
-    )]
-    pub open_orders: AccountLoader<'info, OpenOrdersAccount>,
+    #[account(mut)]
+    /// CHECK: openorders will be checked by openbook_v2
```

Comment:
Sorry, need to comment on this again!

Yes, the openbook v2 cpi will verify that this openorders account is for the given openbook market.

However, the key point is that mango will need to verify that this open_orders account is owned by the mango account passed above. That's essential - otherwise a user could pass in another user's openorders account and do something with it.